spec/gitpoints/points_spec.rb

```diff
@@ -0,0 +1,9 @@
+require "spec_helper"
+
+describe GitPoints::UserPoints do
+  describe ".git_points" do
+    it "returns the points of the user" do
+      expect(git_points("test_user")).to eq 50
```

Comment:
Well, sure, it's a method on an instance of `UserPoints`, and you haven't defined such an instance. I'm not sure what this class is for, but I imagine you'd like to test code to look something like:

```ruby
expect(GitPoints::UserPoints.new(git_points: 50).git_points).to eq 50
```